src/js/pages/jobs/JobsTab.js

```diff
@@ -0,0 +1,65 @@
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {RouteHandler} from 'react-router';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import AlertPanel from '../../components/AlertPanel';
+import DCOSStore from '../../stores/DCOSStore';
+
+class JobsTab extends mixin(StoreMixin) {
+
+  constructor() {
+    super();
+
+    this.store_listeners = [
+      {name: 'dcos', events: ['change']}
+    ];
+  }
+
+  /* eslint-disable no-unused-vars */
+  getContents(id) {
+  /* eslint-enable-no-unused-vars */
+    // Render loading screen
+    if (!DCOSStore.dataProcessed) {
+      return (
+        <div className="container container-fluid container-pod text-align-center
+            vertical-center inverse">
+          <div className="row">
+            <div className="ball-scale">
+              <div />
+            </div>
+          </div>
+        </div>
+      );
+    }
+
+    if (this.props.params.taskID) {
+      return (
+        <RouteHandler />
+      );
+    }
+
+    // Render empty panel
+    return (
+      <AlertPanel
+        title="No Jobs Found"
+        iconClassName="icon icon-sprite icon-sprite-jumbo
```

Comment:
Mmm, OK. But I've seen `className`s newlining like very often – you OK with me leaving it this way?